charts/seed-bootstrap/charts/fluentd-es/templates/fluent-bit-configmap.yaml

```
@@ -382,19 +382,21 @@ data:
     [PARSER]
         Name        addonmanagerParser
         Format      regex
-        Regex       ^(?<severity>.[^:]*):\s(?<log>.*)
+        Regex       ^(?<severity>[A-Z]+):\s(?<log>.*)
 
   modify_severity.lua: |-
     function cb_modify(tag, timestamp, record)
```

Thank you for the PR. My suggestion is:
```suggestion
function cb_modify(tag, timestamp, record)
  local severity = record["severity"]
  if severity == nil or severity == "" then
    return 0, 0, 0
  end

  severity = trim(severity):upper()
  local modified = false

  if severity == "I" or severity == "INF" or severity == "INFO" then
    record["severity"] = "INFO"
    modified = true
  elseif severity == "W" or severity == "WRN" or severity == "WARN" or severity == "WARNING" then
    record["severity"] = "WARN"
    modified = true
  elseif severity == "E" or severity == "ERR" or severity == "ERROR" then
    record["severity"] = "ERR"
    modified = true
  elseif severity == "D" or severity == "DBG" or severity == "DEBUG" then
    record["severity"] = "DBG"
    modified = true
  elseif severity == "N" or severity == "NOTICE" then
    record["severity"] = "NOTICE"
    modified = true
  elseif severity == "F" or severity == "FATAL" then
    record["severity"] = "FATAL"
    modified = true
  end

  if not modified then
    return 0, 0, 0
  end

  return 1, timestamp, record
end
```

---

Nice, It's really cleaner and faster. Thanks!! :)